skyportal/handlers/api/group.py

```diff
@@ -138,22 +141,25 @@ def get(self, group_id=None):
         include_single_user_groups = self.get_query_argument(
             "includeSingleUserGroups", False
         )
-        acls = [acl.id for acl in self.current_user.acls]
         info = {}
         info['user_groups'] = list(self.current_user.groups)
-        info['all_groups'] = (
-            Group.query.all()
-            if "System admin" in acls or "Manage groups" in acls
-            else None
-        )
         if (not include_single_user_groups) or (include_single_user_groups == "false"):
```

Comment:
False by default, need to check for true.

```
include_single_user_groups.lower() == 'true'
```

Also, are you going to remove single user group if the argument is set to false for the super user?

Reply:
That's already being done on lines 147-150

Reply:
I don't see the logic to coerce the variable to true/false anywhere; can you point it out again?

Reply:
@acrellin if the parameter isn't provided, it defaults to `False`, so the above condition is `True` (`not include_single_user_groups`), and single user groups are removed. If the parameter is `True` or `"true"` (both truthy), the above evaluates to `False`, and single user groups are not removed. If the parameter is `"false"` or `False`, the above evaluates to `True`, and single user groups are removed.

Reply:
See updated logic (same condition check, though)

Reply:
My concern was the case in which include_single_user_groups is the string "False", which evaluates to True.  A `.lower()` would fix that.

Reply:
Fixed. Thanks for the suggestion